package.json

```diff
@@ -23,7 +24,8 @@
     "ethjs-unit": "^0.1.6",
     "normalize.css": "^7.0.0",
     "numeral": "^2.0.6",
-    "string-loader": "0.0.1"
+    "string-loader": "0.0.1",
+    "unit": "0.0.1-alpha"
```

Comment:
I can't figure out how this package is used.

Reply:
fixed